lib/category_item.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:meals_app/category_meals_screen.dart';
+
+class CategoryItem extends StatelessWidget {
+  final String id;
+  final String title;
+  final Color color;
+
+  const CategoryItem(
+      {super.key, required this.title, required this.color, required this.id});
```

Comment:
As a rule of thumb, we put the constructor arguments all on their own line, giving the last one a trailing comma